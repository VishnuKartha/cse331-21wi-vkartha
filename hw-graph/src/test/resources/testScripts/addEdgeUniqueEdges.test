# tests if the uniqueness of edges is too strong
# Tests that you can add two edges with the same parent and child if they have different labels
# Then tests that you can add two edges with the same label if it has a different parent or child.

# Create a Graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Adds 4 edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2
AddEdge graph1 n2 n1 e2
AddEdge graph1 n2 n1 e1
