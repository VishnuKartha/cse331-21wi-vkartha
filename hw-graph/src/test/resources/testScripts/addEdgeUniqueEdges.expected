# tests if the uniqueness of edges is too strong
# Tests that you can add two edges with the same parent and child if they have different labels
# Then tests that you can add two edges with the same label if it has a different parent or child.

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add 4 edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1
added edge e2 from n2 to n1 in graph1
added edge e1 from n2 to n1 in graph1
